cluster/gce/util.sh

```diff
@@ -1322,6 +1326,11 @@ ${var_name}: ${var_value}
 EOF
     done
   fi
+  if [[ ${ENABLE_CREDENTIAL_SIDECAR:-false} == "true" ]]; then
+    cat >>$file <<EOF
+ENABLE_CREDENTIAL_SIDECAR: $(yaml-quote ${ENABLE_CREDENTIAL_SIDECAR})
```

Comment:
I would lose the if block. Instead just provider the default.
cat >>$file <<EOF
ENABLE_CREDENTIAL_SIDECAR: $(yaml-quote ${ENABLE_CREDENTIAL_SIDECAR:-false})
EOF

Then we should get the correct behavior but the flag is obvious if someone wants to flip it.

Reply:
Done